Forbid renaming arguments of section variables (#14573)
coq/coq: doc/sphinx/language/extensions/arguments-command.rst

         .. exn:: Flag 'rename' expected to rename @name into @name.
            :undocumented:

>          .. exn:: Section variables such as @name may not have renamed arguments.

Reviewer: ```suggestion
         .. exn:: Section variables such as @name may not be renamed.
```

IIUC (and update the code with any changed wording)
